docker/buildkite/docker-compose.yml

```diff
@@ -22,8 +22,8 @@ services:
         aliases:
           - statsd
 
-  cadence:
-    image: ubercadence/server:master-auto-setup
+  temporal:
+    image: temporalio/temporal:master-auto-setup
     ports:
       - "7933:7933"
       - "7934:7934"
```

Comment:
I don't think we need to expose history service to integration tests. If yes, this port need to be changed to `7234`.